### ISSUE TITLE
tox version 2.7.0 no longer works

### ISSUE THREAD

**Issue by pganssle** [dateutil/dateutil#566]
The `minversion` of `tox` is 2.7.0, but it now complains about `{basepython}` (which is weird because that's documented):

```
$ pip install tox==2.8.0
...
$ tox
Traceback (most recent call last):
  File "~/.local/bin/tox", line 11, in <module>
    sys.exit(cmdline())
  File "/usr/lib/python3.6/site-packages/tox/session.py", line 38, in main
    config = prepare(args)
  File "/usr/lib/python3.6/site-packages/tox/session.py", line 26, in prepare
    config = parseconfig(args)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 242, in parseconfig
    parseini(config, inipath)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 790, in __init__
    self.make_envconfig(name, section, reader._subs, config)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 821, in make_envconfig
    res = meth(env_attr.name, env_attr.default)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1014, in getstring
    x = self._replace(x, name=name, crossonly=crossonly)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1038, in _replace
    return Replacer(self, crossonly=crossonly).do_replace(value)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1058, in do_replace
    return self.RE_ITEM_REF.sub(self._replace_match, x)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1091, in _replace_match
    return self._replace_substitution(match)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1131, in _replace_substitution
    val = self._substitute_from_other_section(sub_key)
  File "/usr/lib/python3.6/site-packages/tox/config.py", line 1125, in _substitute_from_other_section
    "substitution key %r not found" % key)
tox.ConfigError: ConfigError: substitution key 'basepython' not found
```

@gaborbernat

**Comment by gaborbernat** [comment]
Hmm that was added part of 2.9.0. My bad 👎

**Comment by gaborbernat** [comment]
bumped minversion in https://github.com/dateutil/dateutil/pull/568/ (yeah was lazy to create another PR for it, sorry)

**Comment by gaborbernat** [comment]
@pganssle by the way see ``tox -av`` for a list of available commands (hence the need for version ``2.9.0+``)

**Comment by pganssle** [comment]
Fixed with merge of #568.
